Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/ManipulatorManager.cpp

```diff
@@ -67,6 +67,11 @@ namespace AzToolsFramework
             return;
         }
 
+        if (m_activeManipulator && m_activeManipulator->GetManipulatorId() == manipulator->GetManipulatorId())
+        {
+            m_activeManipulator = nullptr;
```

Comment:
suggestion: Calling `.reset()` here might be clearer as this is a `shared_ptr` and we want to explicitly show the ownership count is changing